providers/google/tests/system/common/test_quota_project_system.py

```diff
@@ -0,0 +1,129 @@
+"""Tests for Google Cloud quota project functionality."""
```

Comment:
there are no files with prefix "_system" for system tests. For the examples how to write a good system tests, please check other files in this folder.